### PR TITLE
[BEAM-77] IOs module creation, and reorg of IOs

### PR DESCRIPTION
**Pull request by jbonofre**
[BEAM-77] Create a new IOs module, and start to move the different IOs there

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [X] Make sure tests pass via `mvn clean verify`. (Even better, enable 
       Travis-CI for on your fork and ensure the whole test matrix passes).
 - [X] Replace "<Jira issue #>" in the title with the actual Jira issue 
       number, if there is one.
 - [X] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

### CODE REVIEW

File: ios/bigtable/pom.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```
Comment (dhalperi) [inline ios/bigtable/pom.xml]: call it `io`, not `ios`.

`bigtable` goes in the Google Cloud Platform module -- `io.gcp.bigtable`?

File: ios/hdfs/pom.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dhalperi) [inline ios/hdfs/pom.xml]: can you do contrib stuff as a separate PR? I suspect we're going to need to do a lot of work to make tests pass, etc.

File: ios/bigtable/pom.xml (hunk 1 above)
Comment (jbonofre) [inline ios/bigtable/pom.xml]: OK, I tried to "mimic" runners, sdks where we have a 's'. However, no problem to use io instead.

For io.gcp.bigtable, I guess you talk about the package to use, right ?

File: ios/hdfs/pom.xml (hunk 2 above)
Comment (jbonofre) [inline ios/hdfs/pom.xml]: Sure, I planned to do several commits in the same PR, but I can do several PRs, no problem at all.

**Comment by jbonofre** [comment]
I close this PR to open one PR per IO.
